### ISSUE TITLE
No .jar file

### ISSUE THREAD

**Issue by RenniePet** [lerno/naga#16]
In readme.md it says "Download naga.jar and put it in your classpath.", but I think that was from before the project was migrated to GitHub.

**Comment by lerno** [comment]
Yes, if you want to update this, either by creating a jar or changing the text I'd be happy to accept a pull request.
